assembly/src/tokens/mod.rs

```diff
@@ -250,7 +251,7 @@ fn validate_import_path(path: &str, token: &Token) -> Result<AbsolutePath, Parsi
     // path limbs must be separated by "::"
     for limb in path.split(MODULE_PATH_DELIM) {
         // each limb must be a valid label
-        if !is_valid_label(limb) {
+        if PROCEDURE_LABEL_PARSER.parse_label(limb).is_err() {
             return Err(ParsingError::invalid_module_path(token, path));
         }
     }
```

Comment:
Since this is validating import path rather than procedure label, should we use something like `AbsolutePath::try_from()` here? This would also address the `TODO` in the doc comments for this function.

Reply:
> should we use something like `AbsolutePath::try_from()` here?

Absolutely, made the change in 4c8b2aa25c79f066a95fd960b28bf122148102c8